### PR TITLE
Chore/new models alignment

### PR DESCRIPTION
**Pull request by osmaczko**
Adds missing StatusQ adaptation to https://github.com/status-im/StatusQ/pull/719

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]

File: sandbox/demoapp/StatusAppCommunityView.qml [hunk 1]
```diff
@@ -267,15 +267,15 @@ StatusAppThreePanelLayout {
...
+                nickName: model.localNickname
+                userName: model.displayName
+                pubKey: model.pubKey
+                isVerified: model.isVerified
+                isUntrustworthy: model.isUntrustworthy
+                isContact: model.isContact
                 image.source: model.icon
-                image.isIdenticon: model.isIdenticon
-                isOnline: model.onlineStatus
-                ringSettings.ringSpecModel: model.ringSpecModel
```
Comment (noeliaSD) [inline sandbox/demoapp/StatusAppCommunityView.qml:278]: Why are you removing as well the ringSettings???

File: sandbox/demoapp/data/Models.qml [hunk 2]
```diff
@@ -843,73 +843,50 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
...
+            pubKey: "0x043a7ed0e8752236a4688563652fd0296453cef00a5dcddbe252dc74f72cc1caa97a2b65e4a1a52d9c30a84c9966beaaaf6b333d659cbdd2e486b443ed1012cf04"
+            isVerified: true
+            isUntrustworthy: false
+            isContact: true
+            onlineStatus: 1
             icon: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAYAAAAeP4ixAAAAlklEQVR4nOzW0QmDQBAG4SSkl7SUQlJGCrElq9F3QdjjVhh/5nv3cFhY9vUIYQiNITSG0BhCExPynn1gWf9bx498P7/
                             nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2ImYgiNITTlTdG1nUZ5a92VITQxITFiJmIIjSE0htAYQrMHAAD//+wwFVpz+yqXAAAAAElFTkSuQmCC"
-            isIdenticon: true
             isAdmin: false
-            ringSpecModel: [ ListElement {colorId: 13; segmentLength: 5},
```
Comment (noeliaSD) [inline sandbox/demoapp/data/Models.qml:856]: Same comment here than before.. Why are you removing ring settings? Then we cannot test them in the demo app!

Comment (osmaczko) [reply sandbox/demoapp/data/Models.qml:856]: Good point, thanks, I've just aligned the model with what we have in the backend (there colorHash is obtained from pubKey through Utils which are not accessible in StatusQ). I will fix it in followup PR to not make it a blocker.

File: sandbox/controls/ListItems.qml [hunk 3]
```diff
@@ -394,13 +394,13 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
...
-        trustIndicator: StatusContactVerificationIcons.TrustedType.Verified
-        isMutualContact: true
+        pubKey: "0x043a7ed0e8752236a4688563652fd0296453cef00a5dcddbe252dc74f72cc1caa97a2b65e4a1a52d9c30a84c9966beaaaf6b333d659cbdd2e486b443ed1012cf04"
+        isVerified: true
+        isContact: true
         image.source: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAYAAAAeP4ixAAAAlklEQVR4nOzW0QmDQBAG4SSkl7SUQlJGCrElq9F3QdjjVhh/5nv3cFhY9vUIYQiNITSG0BhCExPynn1gWf9bx498P7/
                       nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2ImYgiNITTlTdG1nUZ5a92VITQxITFiJmIIjSE0htAYQrMHAAD//+wwFVpz+yqXAAAAAElFTkSuQmCC"
         image.isIdenticon: true
-        isOnline: true
+        status: 1 // FIXME: use enum
```
[the remark this answers is not in the document]
Comment (osmaczko) [reply sandbox/controls/ListItems.qml:403]: Yes, you are right. I will change it back to `onlineStatus` or `userStatus` in follow-up.

**Review by noeliaSD — APPROVED** [review]

**Pull request merged**